ctapipe/io/containers.py

```diff
@@ -428,6 +428,7 @@ class TelescopePointingContainer(Container):
 class DataContainer(Container):
     """ Top-level container for all event information """
 
+    event_type = Field('data', "Event type")
```

Comment:
what are the options here? Is it data vs calibration event? Or is it observed vs simulated event?

Reply:
Ah that's indeed a good question, maybe this PR makes sense in this context.
https://github.com/cta-observatory/ctapipe/pull/978

By defining an `enum.Enum` or `enum.IntEnum` the choices are somehow obvious...

Reply:
For now, data and calibration can come from eventio. But I was not really pleased with that choice. Physics and specific calibration names would be better I think.  I will change that in eventio once we agree on the names.